lib/IDE/CodeCompletion.cpp

```diff
@@ -4243,30 +4282,37 @@ class CompletionOverrideLookup : public swift::VisibleDeclConsumer {
       if (FD->isAccessor())
         return;
 
+      if (hasInitializerModifier)
+        return;
+
       if (!hasIntroducer || hasFuncIntroducer)
         addMethodOverride(FD, Reason);
       return;
     }
 
     if (auto *VD = dyn_cast<VarDecl>(D)) {
-      if (!hasIntroducer || hasVarIntroducer) {
+      if (hasInitializerModifier)
+        return;
+
+      if (!hasIntroducer || hasVarIntroducer)
```

Comment:
If someone wrote `required var` or `convenience func`, etc. I suggest we still give them var and func override completions.  It's going to be invalid, but they can accept the fixits.  So something like
```c++
if (hasVarIntroducer || (!hasIntroducer && !hasInitializerModifier)) {
```
and likewise for functions.